dagger-common/src/test/java/io/odpf/dagger/common/serde/proto/protohandler/typehandler/DoublePrimitiveTypeHandlerTest.java

```diff
@@ -90,4 +93,50 @@ public void shouldReturnEmptyArrayOnNull() {
         assertEquals(0, ((double[]) actualValues).length);
     }
 
+    @Test
+    public void shouldFetchParsedValueForFieldOfTypeDoubleInSimpleGroup() {
+        Descriptors.FieldDescriptor fieldDescriptor = TestBookingLogMessage.getDescriptor().findFieldByName("cash_amount");
+        GroupType parquetSchema = org.apache.parquet.schema.Types.requiredGroup()
+                .required(DOUBLE).named("cash_amount")
+                .named("TestGroupType");
+        SimpleGroup simpleGroup = new SimpleGroup(parquetSchema);
+        simpleGroup.add("cash_amount", Double.MAX_VALUE);
```

Comment:
Sure, will use a normal value.
I'm not aware of such a practice though. What's the reason for it ?

Reply:
when using extreme values it is very easy to run into issues where because of framework/serde those values might be set in case of invalid values, which will be missed if we use this in our tests (happened once with us, not in this repo). Using a more specific value avoid such corner cases

Reply:
Sure

Reply:
Fixed as part of commit https://github.com/odpf/dagger/pull/121/commits/f106599978047dbfe5e9f30eeb4c5ecec77dbd7c